dhall/src/Dhall/Main.hs

```diff
@@ -540,19 +551,51 @@ noHeaders
 noHeaders i =
     i
 
+data Config = Config
+    { ascii  :: Maybe Bool } deriving Dhall.Generic
```

Comment:
We should probably also add a configuration option for `explain`, since this configuration is likely to remain Haskell-specific

Reply:
I added support for `explain`. Still can't run the tests until I figure out how to get Nix working on Catalina.